### PR TITLE
Jaeger MK2 PR return of spriter sanity

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
image
image
tl;dr
revert to manual colors, map colors only, coloring isn't gone
removal of arms/legs/visor for jaeger entirely
xenonauten resprite to fit new map variants
jaeger de-legoized

## Why It's Good For The Game
Better maintainability compared to hyperscale on the spriter end, better overall look than grayscale. Hyperscale was made in mind with the tool to amke it sane to use that never came out, and lego pieces are hard to maintain as well, and aren't used. 
The mass of colors makes the marine force look like power rangers, this alleviates that overall, and makes the art style more consistent and in line with itself.
This will probably cause most loadouts to implode due to the removal of the XM02, but such is life.
## Changelog
:cl:
add:New map variants colors for Jaeger/Xenonauten.
del: arm/leg pieces for Jaeger are gone entirely, as Jaeger has been de-leogized, every exo is one set.
del: All Jaeger colors other than map variants.
del: Jaeger XM-02, a few jaeger armorsets that weren't resprited with the update to the sprites.
/:cl:

### CODE REVIEW

**Comment by RaidarV** [comment]
salt pr ? :trollface:

**Review by Kuro020 — APPROVED** [review]
RIP bozo

**Comment by Tailiik** [comment]
Visor color is gone too? But muh ODST cosplay requires blue visor

**Comment by Kuro020** [comment]
image
EVA skull not in the picture because I'm lazy

**Comment by Kuro020** [comment]
image
I forgor EOD 💀

File: code/datums/greyscale/greyscale_configs.dm [hunk 1]
```diff
@@ -35,141 +35,6 @@
...
-
-/datum/greyscale_config/modulararms/helljumper
-	icon_file = 'icons/mob/modular/helljumper.dmi'
-
-/*
- * MODULAR LEGS
- */
-/datum/greyscale_config/modularlegs
-	icon_file = 'icons/mob/modular/infantry.dmi'
-	json_config = 'code/datums/greyscale/json_configs/modularlegs.json'
```
Comment (TiviPlus) [inline code/datums/greyscale/greyscale_configs.dm:123]: json config removal?

**Review by TiviPlus — CHANGES_REQUESTED** [review]
I dont see emissibles on the helmets

**Review by Lumipharon — CHANGES_REQUESTED** [review]
Fix the helmet variant names in the DMI as mentioned and the json files tivi mentioned, otherwise everything is surprisingly ok

**Comment by Lumipharon** [comment]
Kuro says add the no hair flag. 
image

**Comment by Tupinambis** [comment]
So I like the sprites but I think the names are a bit crap.
I'd format it roughly as [Armor Class] [Aesthetic] [Type]

Ex: Light Scout Exosuit
Medium EVA Exosuit
Heavy Assault Exosuit
Xenonautin -> Light/Medium/Heavy Armor Vest
Scout Helmet
Xenonautin -> Open/Closed Combat Helmet 
Rownin -> Combat Harness 

Easy readability good, "Jaeger Skirmisher light exo" not even telling you the armor type until the third word is bad, especially for reading down a list.

**Comment by Lumipharon** [comment]
Closing as I'm opening the new one.
